auth_style/templates/allauth/elements/panel.html

```diff
@@ -1,5 +1,5 @@
 {% load allauth %}
-<div class="card card-compact glass bg-base-300">
+<div class="border card card-compact bg-base-100 border-[rgba(0,0,0,.18)] dark:border-[rgba(255,255,255,.3)]">
```

Comment:
Is it possible to make these colors less magical? Maybe use this: https://tailwindcss.com/docs/colors#adjusting-opacity